ui/plugins/annotations_panel.jsx

```diff
@@ -0,0 +1,152 @@
+                  onClick={deleteBugEvent}
+                  title={`Delete relation to bug ${props.bug.bug_id}`}>
+            <i className="fa fa-times-circle"></i>
+            </span>
+      </span>
+    );
+};
+
+
+const RelatedBug = props => (
+    <span>
+        <p className="annotations-bug-header font-weight-bold">Bugs</p>
+        <ul className="annotations-bug-list">
+            {props.bugs.map((bug, index) =>
+            <li key={index} >
+                <RelatedBugSaved bug={bug} getBugUrl={props.getBugUrl} deleteBug={props.deleteBug} />
+            </li>)}
+        </ul>
+    </span>
+);
+
+
+function TableRow(props) {
+
+    const deleteEvent = () => {
+        props.deleteClassification(props.classification);
+    };
+
+    const failureId = props.classification.failure_classification_id;
+    let iconClass = failureId === 7 ? "fa-star-o" : "fa fa-star";
+    const classificationName = props.classificationTypes.classifications[failureId];
+
+    return (
+        <tr>
+            <td>
+                {props.dateFilter(props.classification.created, 'EEE MMM d, H:mm:ss')}
+            </td>
+            <td>
+                {props.classification.who}
+            </td>
+            <td>
+                {/* TODO: the classification label & star has been used in the job_details_pane.jxs
+                 so it should probably be made its own component when we start using import */}
+                <span title={classificationName.name}><i className={`fa ${iconClass}`}></i>
+                <span className="ml-1">{classificationName.name}</span></span>
+            </td>
+            <td>
+                {props.classification.text}
+            </td>
+            <td>
+                <span onClick={deleteEvent}
+                    className="classification-delete-icon hover-warning pointable"
+                    title="Delete this classification">
+                    <i className="fa fa-times-circle"></i>
+                </span>
+            </td>
+        </tr>
+    );
+};
+
+
+function AnnotationsTable(props) {
+    return (
+        <table className="table-super-condensed table-hover">
+            <thead>
+                <tr><th>Classified</th><th>Author</th><th>Classification</th><th>Comment</th></tr>
+            </thead>
+            <tbody>
+                {props.classifications.map((classification, index) =>
+                    <TableRow
+                            key={index} dateFilter={props.dateFilter} classification={classification}
+                            deleteClassification={props.deleteClassification}
+                            classificationTypes={props.classificationTypes} />)}
+            </tbody>
+        </table>
+    );
+};
+
+
+class AnnotationsPanel extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        // TODO: move this to ui/js/constants.js when we start using import;
```

Comment:
You should be able to do this now, after a rebase.

Reply:
Just realized this won't work as a constant because it needs access to props and the $injector.